distribution/docs/src/main/resources/content/_configuring/importing-exported-configurations-contents.adoc

```diff
@@ -6,20 +6,21 @@
 
 ==== Importing the Exported Configuration Settings
```

Comment:
❗️  Same comment as above. This section comes right after the "Exporting/Importing Configuration Settings" section and they both lay out the exact same steps. One of the import sections should be removed.

Reply:
Same reply

Reply:
❓  What was the justification for having a duplicated section?